index.html

```diff
@@ -28,9 +28,152 @@ <h4 class="connect"> LET’S CONNECT</p>
                 <li><a href="#"><img src="./image/Linkedin.svg" alt=""></a></li>
             </ul>
             </div>
-
 </section>
 </section>
+<section class="Workssection">
+    <div class="cardBack">
+    <a class="card1" href="#"><img src="./image/Snapshoot Portfolio1 (1).png" alt=""></a>
+            <h3  class="tonic" >Tonic</h3>
+    
+    <div>
+    <ul class="works">
+        <li class="canopy"> CANOPY</li>
+        <li class="BackendDev">BackendDev</li>
+        <li class="num"> 2015 </li>
+     </ul>
+    <p class="section1"> A daily selection of privately personalized reads; no accounts or sign-ups required </p>
+</div>
+
+    <div class="works2">
+        <div>html</div>
+        <div>css</div>
+        <div>javascript</div>
+   </div>
+   <div class="buttondir">
+    <button class="buttonProject"  type="button" class="resume-btn">See Project</button>
```

Comment:
- [ ] Kindly consider the color as used on the figma
### Figma
![image](https://user-images.githubusercontent.com/86200356/199233248-4de7845f-bf84-421b-ab08-7b2b905fd3ea.png)

### Your Design
![image](https://user-images.githubusercontent.com/86200356/199233312-473a4136-79fc-4e41-a229-a20332ef60e9.png)